src/Helper/Translation/Translator.php

```diff
@@ -32,7 +32,7 @@ public function addCatalogues(): void
 
         if ($environment->isLocalPulled()) {
             foreach ($environment->getLocal()->getTranslations() as $file) {
-                $this->translator->addResource($file->format, $file->resource, $file->locale, $file->domain);
+                $this->translator->addResource($file->format, $file->resource, $file->locale, $environment->getName());
```

Comment:
Here also, my instinct telling me to use the alias

Reply:
Yes also thought about it, but the [ClientRequest](https://github.com/ems-project/EMSClientHelperBundle/blob/release/3.6/src/Helper/Elasticsearch/ClientRequest.php) getCacheKey uses the environment name. Could not just switch to alias without testing more ..., maybe in a next release